elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -10,23 +10,34 @@ $LOCAL-VARIABLES: (
 	MinWidth: 50vw,
 	MaxWidth--mobile: 94vw,
 	MaxHeight: 90vh,
-	Padding: calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 3.5) calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2),
+	Padding: 
+		calc(#{pfe-var(container-padding)} * 2) 
+		calc(#{pfe-var(container-padding)} * 3.5) 
+		calc(#{pfe-var(container-padding)} * 2) 
+		calc(#{pfe-var(container-padding)} * 2),
 	overlay: (
-		BackgroundColor: pfe-var(overlay)
+		BackgroundColor: pfe-var(overlay),
+		Overflow: auto,
+	),
+	window: (
+		BackgroundColor: pfe-var(surface--lightest)
 	),
 	close: (
-		size: calc(#{pfe-var(ui--element--size)} - 4px)
+		size: calc(#{pfe-var(ui--element--size)} - 4px),
+		Padding: pfe-var(container-padding),
+		BackgroundColor: #fff,
 	)
 );
+//@include pfe-local-debug;
 
 :host {
 	display: block;
 	position: relative;
 
    @media screen and (max-height: $pfe-modal--breakpoint--medium) {
-	   @include pfe-print-local((
-		   Padding: 0 0 0 calc(#{pfe-var(container-padding)} * 2)
-	   ));
+	   Padding: pfe-local( Padding, 
```

Comment:
I think this change is what's causing the trigger to get padding added to it when the screen height changes.

![pfe-modal-padding](https://user-images.githubusercontent.com/330256/91046033-c1ce0f80-e5e5-11ea-8a2d-1c3d61719c41.gif)

Reply:
This should print the variable - we don't want to apply padding to the pfe-modal tag, but rather, to the pfe-modal content in the shadow DOM.

Reply:
thanks, fixed!